pinot-controller/src/test/resources/memory_estimation/schema-with-metadata__dateTimeFieldSpec.json

```diff
@@ -5,34 +5,34 @@
       "dataType": "INT",
       "name": "colInt",
       "singleValueField": false,
-      "cardinality": 500,
+      "cardinality": 50,
```

Comment:
Let's not change these as discussed earlier. Let's make data generation deterministic

Reply:
Done.